package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "bottlenose",
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "BABEL_ENV=build lerna bootstrap",
     "build": "lerna exec --parallel 'BABEL_ENV=build babel src --out-dir build --source-maps --extensions .js --config-file ../../babel.config.js --delete-dir-on-start --no-comments'",
     "package:create": "lerna create --dependencies rxjs --license MIT",
     "lint": "lerna exec yarn lint",
```

Comment:
This bootstrap command is important.  

It links packages from the top-level `node_modules` in the monorepo to the sub-packages.  That way certain dependencies like `babel` can be installed once and used in all of the packages.

It also allows sub-modules to reference each other.  So, for example, the `rxtranscribe` (speech-to-text) module uses the `rxws` (web socket) module.  Bootstrapping allows them to reference one another locally without releasing them to npm when we're developing.

Reply:
For the most part, packages in the `./packages` directory are normal npm packages.  But there are a few monorepo commands that are sometimes needed...  The lerna commands I end up using a lot are these:
```bin
# build the modules and their dependencies
yarn lerna bootstrap
# build the modules
yarn build
# install an npm module to a package
yarn lerna add new-module --scope @bottlenose/my-package
# run tests for a specific package
cd ./packages/my-package && yarn test
# or
yarn lerna test --scope @bottlenose/my-package
# create a new package (this will require some changes to the auto-generated package.json`)
yarn lerna create @bottlenose/my-new-package
```
I'll add this to the `docs/contributors/README.md` too, which will hopefully have enough information to explain what's going on to a new contributor...